app/models/PaymentMode.scala

```diff
@@ -0,0 +1,18 @@
+package models
+
+sealed trait PaymentMode {
```

Comment:
I keep meaning to try https://github.com/lloydmeta/enumeratum for this kind of thing.